registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/abstractverticle/WorkflowInternalActionDTO.java

```diff
@@ -43,8 +38,8 @@ public WorkflowInternalActionDTO() {
 	/** The resume remove tags. */
 	private String resumeRemoveTags;
 
-
-
+	/** The sub process. */

```

Comment:
Change the comment also to additional info process

Reply:
fixed